invenio_rdm_records/fixtures/communities.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Communities fixture module."""
+
+from pathlib import Path
+
+from invenio_communities.fixtures.tasks import create_demo_community
+
+from .fixture import FixtureMixin
+
+
+class CommunitiesFixture(FixtureMixin):
+    """Communities fixture."""
+
+    def __init__(self, logo_path, search_paths, filename):
```

Comment:
minor: Can "logo path" be moved to the end of the params list? to keep consistency of API with other `Fixture` classes.